stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/gallery/internal/AttachmentGalleryVideoPageFragment.kt

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.gallery.internal
+
+import android.content.Context
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.MediaController
+import android.widget.Toast
+import androidx.core.view.updateLayoutParams
+import androidx.fragment.app.Fragment
+import com.getstream.sdk.chat.images.load
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.ui.ChatUI
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamUiItemAttachmentGalleryVideoBinding
+import io.getstream.chat.android.ui.gallery.AttachmentGalleryViewMediaStyle
+import io.getstream.chat.android.ui.gallery.options.AttachmentGalleryOptionsViewStyle
+
+internal class AttachmentGalleryVideoPageFragment : Fragment() {
+
+    private var _binding: StreamUiItemAttachmentGalleryVideoBinding? = null
+    private val binding get() = _binding!!
+
+    /**
+     * Holds the style necessary to stylize the play button.
+     *
+     * Fetching the style depends on [Context] so use this property
+     * only after it has been obtained during or after [onAttach].
+     */
+    private val style by lazy {
+        AttachmentGalleryViewMediaStyle(
+            context = requireContext(),
+            attrs = null
+        )
+    }
```

Comment:
Same here, should work only if Context is wrapped `requireContext().createStreamThemeWrapper()`.

```
    <style name="StreamUiTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
       ...
        <item name="streamUiAttachmentGalleryVideoAttachmentsStyle">@style/StreamUi.AttachmentGallery.MyCustomVideoAttachments</item>
    </style>

    <style name="StreamUi.AttachmentGallery.MyCustomVideoAttachments">
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoButtonIcon">@drawable/stream_ui_ic_audio</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconElevation">3dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconPaddingTop">2dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconPaddingBottom">2dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconPaddingStart">3dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconPaddingEnd">2dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconCornerRadius">24dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayVideoIconBackgroundColor">@color/stream_ui_literal_white</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayIconWidth">45dp</item>
        <item name="streamUiAttachmentGalleryViewMediaPlayIconHeight">45dp</item>
        <item name="streamUiAttachmentGalleryViewMediaImagePlaceholder">@drawable/stream_ui_picture_placeholder</item>
    </style>
```